test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -76,6 +76,12 @@ public void All_props_and_targets_add_themselves_to_MSBuildAllTargets()
                 return;
             }
 
+            //  Disable this test when using full Framework MSBuild, as the paths to the props and targets are different
+            if (UsingFullFrameworkMSBuild)
```

Comment:
I would much rather we create a custom attribute and skip this dynamically: https://github.com/xunit/samples.xunit/blob/master/DynamicSkipExample